### ISSUE TITLE
VCV RACK port?

### ISSUE THREAD

**Issue by Eurikon** [ryukau/VSTPlugins#2]
Hi, could you please consider porting your plugins such as the cymbal wave etc. to VCV RACK module platform?

https:///vcvrack.com

**Comment by ryukau** [comment]
Hi, Eurikon.

I haven't used VCV RACK, so I'm not planning to do it.

If you are willing to port it by yourself, all of the DSP related codes are in `<PluginName>/source/dsp/*` and parameters are in `<PluginName>/source/parameter.*`. Main processing function is `DSPCore.process()` in `dspcore.hpp`.

**Comment by RustoMCSpit** [comment]
hi @ryukau , the cardinal team would require the code to be relicensed to gpl3 or later for this process to be started

**Comment by ryukau** [comment]
@RustoMCSpit No, I can't. GPLv3 is inherited from VST 3 SDK. Below is a link of their documentation about license.

- [What are the licensing options - VST 3 Developer Portal](https://steinbergmedia.github.io/vst3_dev_portal/pages/VST+3+Licensing/What+are+the+licensing+options.html#open-source-gplv3-license)

It only says "GPLv3". It does not say "GPLv3+".

**Comment by RustoMCSpit** [comment]
https://community.vcvrack.com/t/uhhyou-plugins-module/22076

**Comment by RustoMCSpit** [comment]
https://github.com/steinbergmedia/vst3sdk/issues/126
